HCK01_2022_Virtual/README.md

```diff
@@ -121,9 +238,28 @@ In future hackathons, we will focus on:
 
 [How to add a new reproducible registration processing pipeline?](./ReproducibleResource/README.md)
 
+### Challenge Dataset 1: fMOST Mouse Brain Registration to CCFv3
+
+The fMOST brain volumes and CCFv3 atlas for the hackathon are
+available [on the BIL here](https://download.brainlib.org/hackathon/2022_GYBS/data/). The input is a
+single fMOST NIFTI brain volume.
+
+### Challenge Dataset 1: Light-Sheet Imaged Mouse Brain Registration to CCFv3
```

Comment:
1 -> 2

Reply:
resolved in https://github.com/InsightSoftwareConsortium/GetYourBrainStraight/pull/22/commits/fc96c8c7852339bc94e75f55912e2af4ef33702b